src/BuildConfig.cc

```diff
@@ -97,6 +97,68 @@ BuildConfig::BuildConfig(const std::string& packageName, const bool isDebug)
   }
 }
 
+static std::vector<std::string>
+parseEnvFlags(const std::string& env) {
```

Comment:
This function is hard to follow; please make this clearer and add more comments.

Reply:
Added detailed comments in the latest commit. Also changed the code a little bit for some edge cases.